supernode/plugins/plugins.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package plugins
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/dragonflyoss/Dragonfly/supernode/config"
+)
+
+var mgr = NewManager()
```

Comment:
Shall we have the default value in case that the user does not provide a configuration file.

Reply:
Could we to do this in every concrete plugin manager to return a default plugin implementation, such as `store.Manager.Get`? Because that the package `plugins` cannot the default implementation of every plugin type.